torch/fx/experimental/param_fetch.py

```diff
@@ -0,0 +1,42 @@
+from torch.fx.graph_module import GraphModule
+from typing import Any, Dict, List, Tuple
+import torch.nn as nn
+
+
+"""This dict maps the nn.Module class name to the attribute name list that we want to fetch for lowering.
+The first integer in the tuple is the version number of the nn.Module class when we create the parameter list.
+If there's a version mismatch then it means the parameter names in the book might be mismatched with nn.Module.
+"""
+module_fetch_book: Dict[str, Tuple[int, List[str]]] = {
+    "Linear": (1, ["weight", "bias"]),
+    "Conv2d": (1, ["weight", "bias", "kernel_size", "stride", "padding", "dilation", "groups", "padding_mode"]),
+    "BatchNorm2d": (2, ["weight", "bias", "running_mean", "running_var"]),
+}
+
+def fetch_module(mod: nn.Module) -> Dict[str, Any]:
```

Comment:
"Fetch module" implies you're going to return a `Module` instance, but you're not. This is probably better named something like `extract_attrs_for_lowering`